spec.emu

```diff
@@ -683,6 +684,7 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
                 1. Perform ! _env_.InitializeBinding(_in_.[[LocalName]], _namespace_).
               1. <ins>Else if _in_.[[ImportName]] is ~module~, then</ins>
                 1. <ins>Let _moduleSourceObject_ be _importedModule_.[[ModuleSourceObject]].</ins>
+                1. <ins>If _moduleSourceObject_ is *null*, throw a *SyntaxError* exception.</ins>
```

Comment:
How would it be possible for this to ever happen do you think? Or is this more of a guard?

Reply:
This can happen if you do
```js
import module m from "./file.js";
```